src/services/exports/exports.repository.ts

```diff
@@ -730,6 +730,44 @@ const generateCsvListeGestionnaires = async (gestionnaires, res: Response) => {
   }
 };
 
+const generateCsvHistoriqueDossiersConvention = async (
+  structures: any[],
+  res: Response,
+) => {
+  try {
+    const fileHeaders = [
+      'Id de la structure',
+      'Nom de la structure',
+      'Date de la demande',
+      'Date de fin du prochain contrat',
+      'Nombre de postes',
+      'Type de la demande',
+    ];
+
+    res.write(
+      [
+        fileHeaders.join(csvCellSeparator),
+        ...structures.map((structure) =>
```

Comment:
Normalement pas besoin mais 
```suggestion
        ...structures?.map((structure) =>
```

Reply:
erreur sentry si on met le ?

Reply:
ca faisait quoi comme erreur Sentry ? bizarre que ca puisse planter ou alors c'est lié à la destructuration ? Ci c'est le cas et que structures est null / undefined ca planterait non ?